libs/coin-modules/coin-xrp/src/cli-transaction.ts

```diff
@@ -0,0 +1,11 @@
+/* istanbul ignore file: untested because empty */
```

Comment:
Why didn't you keep the content of the old `cli-transaction` file?

Reply:
It's just me doing a bad copy/paste of the EVM coin-module... (And I'm wondering now why this one is empty...) 🤦‍♂️

Reply:
Added the file back with some modifications for typing and removal of the lodash dep 👍